requirements.txt

```diff
@@ -1,6 +1,9 @@
-matplotlib
-numpy
-scipy
+matplotlib==3.2.1
+numpy==1.18.1
+scipy==1.4.1
```

Comment:
Do we really need to define the requirements that tight?

Reply:
true, I've instead updated the requirements with >= instead of == so that we are not that tight but at the same time have the benefit of knowing what versions should work. I think it's really important that we do nevertheless specify at a minimum the combination of package versions that we know work because we all seem to be using different versions of each packages within the team.  So it makes it harder to know if the issue is coming from the version packages or something else